test/utils/nodes/nodes.go

```diff
@@ -101,3 +101,11 @@ func MatchingOptionalSelectorPTP(toFilter []ptpv1.NodePtpDevice) ([]ptpv1.NodePt
 	}
 	return res, nil
 }
+
+func IsSingleNode() (bool, error) {
```

Comment:
Either move to the cluster package or rename it

Reply:
Renamed